scripts/build-packages.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env zsh
+# this cookie was getting too long for a JSON prop
+# builds the Phoenix yavascripst from src to lib
+
+# die on error
+setopt -e
+
+for pkg in packages/*/ ; do
+  babel ${pkg}/src --out-dir ${pkg}/build
```

Comment:
it'd be nicer imo if this ran `npm run build` or the equivalent in each folder, so it can delegate to its npm run-scripts